brokerapi/brokers/cloudsql/common-definition.go

```diff
@@ -17,167 +17,198 @@ package cloudsql
-	},
-	{
-		FieldName: "disk_size",
-		Type:      broker.JsonTypeString,
-		Details:   "In GB (only for 2nd generation instances).",
-		Default:   "10",
-	},
-	{
-		FieldName: "region",
-		Type:      broker.JsonTypeString,
-		Details:   "The geographical region.",
-		Default:   "us-central",
-	},
-	{
-		FieldName: "zone",
-		Type:      broker.JsonTypeString,
-		Details:   "(only for 2nd generation instances)",
-	},
-	{
-		FieldName: "disk_type",
-		Type:      broker.JsonTypeString,
-		Details:   "(only for 2nd generation instances)",
-		Default:   "ssd",
-	},
-	{
-		FieldName: "failover_replica_name",
-		Type:      broker.JsonTypeString,
-		Details:   "(only for 2nd generation instances) If specified, creates a failover replica with the given name.",
-		Default:   "",
-	},
-	{
-		FieldName: "maintenance_window_day",
-		Type:      broker.JsonTypeString,
-		Details:   "(only for 2nd generation instances) The day when disruptive updates (updates that require an instance restart) to this CloudSQL instance can be made. Day of week (1-7), starting on Monday.",
-		Default:   "1",
-	},
-	{
-		FieldName: "maintenance_window_hour",
-		Type:      broker.JsonTypeString,
-		Details:   "(only for 2nd generation instances) The hour of the day when disruptive updates (updates that require an instance restart) to this CloudSQL instance can be made. Hour of day 0-23.",
-		Default:   "0",
-	},
-	{
-		FieldName: "backups_enabled",
-		Type:      broker.JsonTypeString,
-		Details:   "Should daily backups be enabled for the service?",
-		Default:   "true",
-	},
-	{
-		FieldName: "backup_start_time",
-		Type:      broker.JsonTypeString,
-		Details:   "Start time for the daily backup configuration in UTC timezone in the 24 hour format - HH:MM.",
-		Default:   "06:00",
-	},
-	{
-		FieldName: "binlog",
-		Type:      broker.JsonTypeString,
-		Details:   "Whether binary log is enabled. If backup configuration is disabled, binary log must be disabled as well. Defaults: `false` for 1st gen, `true` for 2nd gen, set to `true` to use.",
-	},
-	{
-		FieldName: "activation_policy",
-		Type:      broker.JsonTypeString,
-		Details:   "The activation policy specifies when the instance is activated; it is applicable only when the instance state is RUNNABLE.",
-		Default:   "ON_DEMAND",
-		Enum: map[interface{}]string{
-			"ALWAYS":    "Always, instance is always on.",
-			"NEVER":     "Never, instance does not turn on if a request arrives.",
-			"ON_DEMAND": "On Demand, instance responses to incoming requests and turns off when not in use.",
+func commonBindComputedVariables() []varcontext.DefaultVariable {
+	return []varcontext.DefaultVariable{
+		// legacy behavior dictates that empty values get defaults
+		{Name: "password", Default: `${password == "" ? rand.base64(32) : password}`, Overwrite: true},
```

Comment:
Instead of repeating your config macro strings here, consider defining them as constants.